lib/secure-boot-lib.robot

```diff
@@ -218,3 +218,26 @@ Execute File In UEFI Shell
     Press Enter
     ${out}=    Read From Terminal Until    FS0:\\>
     RETURN    ${out}
+
+Remove Old Secure Boot Keys
```

Comment:
@m-iwanicki lets add `in OS` to the names of those keywords so we could tell that e.g. we are doing `Remove Old Secure Boot Keys in OS` because `Remove Old Secure Boot Keys` could be also done from BIOS menu

Reply:
added in [5b71c47](https://github.com/Dasharo/open-source-firmware-validation/pull/161/commits/5b71c477c2d2bf39168f700972f6a9e92a46964c)